tests/pages/catalog/packages/PackagesTab-cy.js

```diff
@@ -165,4 +165,24 @@ describe("Packages Tab", function() {
       });
     });
   });
+
+  context("respect DCOS version", function() {
+    beforeEach(function() {
+      cy
+        .route(/dcos-version/, {
+          version: "1.6",
+          "dcos-image-commit": "null",
+          "bootstrap-id": "null"
+        })
+        .visitUrl({ url: "/catalog", logIn: true });
+    });
+
+    it("cant install package because of DCOS version", function() {
```

Comment:
I am assuming somewhere else there is a test for the button being enabled. If not that would be nice to have the other case tested.

Also this logic here has quite some possible inputs. 
```
+      MetadataStore.version &&
+      cosmosPackage.minDcosReleaseVersion &&
+      semver.compare(
+        semver.coerce(MetadataStore.version),
+        semver.coerce(cosmosPackage.minDcosReleaseVersion)
+      ) < 0
```

I would have unit tested that logic.